tplinkrouterc6u/dataclass.py

```diff
@@ -30,4 +30,10 @@ class Status:
     guest_5g_enable: bool
     wifi_2g_enable: bool
     wifi_5g_enable: bool
+    wan_ipv4_uptime: int
+    mem_usage: float
+    cpu_usage: float
+    cpu1_usage: float
```

Comment:
`cpu1_usage`, `cpu2_usage` and `cpu3_usage` are not needed as you calculate it in `cpu_usage`